travis_configure.sh

```diff
@@ -27,13 +27,28 @@ print_error_msg ()
 # linux:
 #   deb:
 #     - fpm
+# osx:
+#   - sed, (libtool), openssl, mysql
 # Returns 0
 before_install()
 {
     if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
         if [[ "$DIST_PACKAGE_TARGET" == "DEB" ]]; then
             gem install fpm
         fi
+    elif [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+        brew install gnu-sed
+        xcode_version=`xcodebuild -version | grep 'Xcode' | cut -f 2 -d ' '`
+        ## reinstall libtool on osx_image xcode6.x to ensure gsed is found by
+        #  glibtoolize
+        if [[ "$xcode_version" == 6.*  ]]; then
```

Comment:
Fair enough.
Anyway there is a double space after the wildcard/pattern matching